client/keys/delete.go

```diff
@@ -47,7 +62,7 @@ func runDeleteCmd(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	fmt.Println("Password deleted forever (uh oh!)")
+	fmt.Fprintln(os.Stderr, "Password deleted forever (uh oh!)")
```

Comment:
Shouldn't this be "key deleted forever"? I don't think we're deleting a password...

Reply:
Correct, amended.